router/userRouter.js

```diff
@@ -11,10 +11,56 @@ import registerRules from "../validation/registerRules.js";
 
 const router = express.Router();
 
-router.get("/", getUsers);
-router.post("/add", validate(registerRules), addUser);
-router.post("/login", loginUser);
-router.put("/update", updateUser);
-router.delete("/delete", deleteUser);
+// router.get("/", getUsers);
+// router.post("/add", validate(registerRules), addUser);
+// router.post("/login", loginUser);
+// router.put("/update", updateUser);
+// router.delete("/delete", deleteUser);
+
```

Comment:
I assume that all of the routes originally start with /user/ because the name of this file is userRouter :)

Reply:
That is correct. In server.js it's: `server.use("/users", userRouter);`.
But that's only the case for these line 14 to 18 that i commented out. The endpoint list after that, was supposed to start all endpoints at `/`. I put them all in the file userRouter (which is a bit confusing, i know) to have them all together in one file.